projects/coredns/coredns/1-22/patches/0001-Bump-golang-x-net-and-golang-x-text-dependency.patch

```diff
@@ -0,0 +1,298 @@
+-	golang.org/x/sys v0.0.0-20211216021012-1d35b9e2eb4e
++	golang.org/x/sys v0.5.0
+ 	google.golang.org/api v0.64.0
+ 	google.golang.org/grpc v1.43.0
+ 	gopkg.in/DataDog/dd-trace-go.v1 v1.34.0
+@@ -40,3 +40,72 @@ require (
+ 	k8s.io/client-go v0.23.1
+ 	k8s.io/klog/v2 v2.40.1
+ )
++
++require (
++	cloud.google.com/go v0.99.0 // indirect
++	github.com/Azure/go-autorest v14.2.0+incompatible // indirect
++	github.com/Azure/go-autorest/autorest/adal v0.9.18 // indirect
++	github.com/Azure/go-autorest/autorest/azure/cli v0.4.2 // indirect
++	github.com/Azure/go-autorest/autorest/date v0.3.0 // indirect
++	github.com/Azure/go-autorest/autorest/to v0.2.0 // indirect
++	github.com/Azure/go-autorest/logger v0.2.1 // indirect
++	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
++	github.com/DataDog/datadog-go v4.4.0+incompatible // indirect
++	github.com/DataDog/sketches-go v1.0.0 // indirect
++	github.com/Microsoft/go-winio v0.5.0 // indirect
++	github.com/beorn7/perks v1.0.1 // indirect
++	github.com/cespare/xxhash/v2 v2.1.1 // indirect
++	github.com/coreos/go-semver v0.3.0 // indirect
++	github.com/coreos/go-systemd/v22 v22.3.2 // indirect
++	github.com/davecgh/go-spew v1.1.1 // indirect
++	github.com/dimchansky/utfbom v1.1.1 // indirect
++	github.com/evanphx/json-patch v4.12.0+incompatible // indirect
++	github.com/flynn/go-shlex v0.0.0-20150515145356-3f9db97f8568 // indirect
++	github.com/go-logr/logr v1.2.0 // indirect
++	github.com/gogo/protobuf v1.3.2 // indirect
++	github.com/golang-jwt/jwt/v4 v4.0.0 // indirect
++	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
++	github.com/google/go-cmp v0.5.6 // indirect
++	github.com/google/gofuzz v1.2.0 // indirect
++	github.com/google/uuid v1.3.0 // indirect
++	github.com/googleapis/gax-go/v2 v2.1.1 // indirect
++	github.com/googleapis/gnostic v0.5.5 // indirect
++	github.com/imdario/mergo v0.3.12 // indirect
++	github.com/jmespath/go-jmespath v0.4.0 // indirect
++	github.com/json-iterator/go v1.1.12 // indirect
++	github.com/mitchellh/go-homedir v1.1.0 // indirect
++	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
++	github.com/modern-go/reflect2 v1.0.2 // indirect
++	github.com/opentracing-contrib/go-observer v0.0.0-20170622124052-a52f23424492 // indirect
++	github.com/oschwald/maxminddb-golang v1.8.0 // indirect
++	github.com/philhofer/fwd v1.1.1 // indirect
++	github.com/pkg/errors v0.9.1 // indirect
++	github.com/prometheus/procfs v0.6.0 // indirect
++	github.com/spf13/pflag v1.0.5 // indirect
++	github.com/stretchr/objx v0.2.0 // indirect
++	github.com/tinylib/msgp v1.1.2 // indirect
++	go.etcd.io/etcd/client/pkg/v3 v3.5.1 // indirect
++	go.opencensus.io v0.23.0 // indirect
++	go.uber.org/atomic v1.7.0 // indirect
++	go.uber.org/multierr v1.6.0 // indirect
++	go.uber.org/zap v1.17.0 // indirect
++	golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4 // indirect
++	golang.org/x/net v0.0.0-20220722155237-a158d28d115b // indirect
```

Comment:
Not sure this will work according to https://go.dev/ref/mod#go-mod-file-replace. 

> Note that a replace directive alone does not add a module to the [module graph](https://go.dev/ref/mod#glos-module-graph). A [require directive](https://go.dev/ref/mod#go-mod-file-require) that refers to a replaced module version is also needed, either in the main module’s go.mod file or a dependency’s go.mod file. A replace directive has no effect if the module version on the left side is not required.